server/routes/request-schema.ts

```diff
@@ -0,0 +1,14 @@
+import { check } from 'express-validator'
+
+const regExp = /^[A-Za-z0-9]*$/
+const schema = [
```

Comment:
Can we change the name of this to something like `prisonerNumberSchema` so it's more clear what it is for? (and export as that name too)